tests/unit/router.tap.js

```diff
@@ -75,3 +75,52 @@ tap.test('koa-router', function tests(t) {
   })
   t.autoend()
 })
+
+tap.test('@koa/router', function tests(t) {
```

Comment:
Ideally, we'd prob have the `@koa/router` stuff split for greater delineation between the modules/versions we are exercising but no reason to hold this up. We can always restructure later as the need arises.